### ISSUE TITLE
Game crashes upon mining flint

### ISSUE THREAD

**Issue by KoloXD** [The-Nexus-Project/Limitless-3#26]
### Describe your issue, what happened?
As said in title, the game crashes if attempting to mine a flint on the ground from Extended Caves.

### To Reproduce
Take pickaxe of choice (was a stone one in my case)
Start hitting away on the flint.

### Crash Reports & Logs
https://pastebin.pl/view/01c8ae9b

### Affected Version
- Pack Version: 1.7.1b
- On a server?: Nope

**Comment by Gbergz** [comment]
Hey, just tried and I can't reproduce this.

GIF: https://gyazo.com/2ba18a83998db2c5174f95870e04bca2

Are you playing with OptiFine?

**Comment by KoloXD** [comment]
I have not edited the mod pack whatsoever, so if optifine is not in it by default then no. Am using MultiMC to handle my modpacks however as the Twitch client is an unresponsive mess, not sure if that affects anything.

Edit: Upon further testing I am almost certain that a 0 durability broken tool is the cause, crashed with both a stone and diamond pick.

**Comment by Khepricus** [comment]
I had a user on a private server I'm running complain of this exact same issue and stack trace when breaking a fern and I couldn't reproduce it either until just now when I tried breaking one with a stone pickaxe I wore down to "Broken" durability. Sure enough, got the exact same crash once I used the broken pick to break a fern.

I haven't tried any picks other than stone, and it only seems to happen in survival mode (once I reloaded and switched to creative it just seems like the fern doesn't actually break) but @KoloXD seems to be right on the money with it being an issue with the broken pick.
